scripts/lib.star

```diff
@@ -831,34 +803,43 @@ def release_canary_npm_packages_step(edition):
             'end-to-end-tests',
         ],
         'environment': {
-            'GITHUB_PACKAGE_TOKEN': {
-                'from_secret': 'github_package_token',
-            },
+            'GITHUB_PACKAGE_TOKEN': from_secret('github_package_token'),
         },
         'commands': [
             './scripts/circle-release-canary-packages.sh',
         ],
     }
 
-def deploy_to_kubernetes_step(edition, is_downstream=False):
+def push_to_deployment_tools_steps(edition, is_downstream=False):
     if edition != 'enterprise' or not is_downstream:
-        return None
+        return []
 
-    return {
-        'name': 'deploy-to-kubernetes',
-        'image': alpine_image,
-        'depends_on': [
-            'build-docker-images',
-        ],
-        'environment': {
-            'CIRCLE_TOKEN': {
-                'from_secret': 'deployment_tools_circle_token',
-            },
+    return [
+        {
+            'name': 'push-to-deployment_tools-config',
+            'image': build_image,
+            'depends_on': [
+                'build-docker-images',
+                # This step should have all the dependencies required for packaging, and should generate
+                # dist/grafana.version
+                'gen-version',
+            ],
+            'commands': [
+                'cat ./deployment_tools_config.json | sed "s/{version}/$(cat ./dist/grafana.version)/g" > ./deployment_tools_config_resolved.json'
+            ]
         },
-        'commands': [
-            './bin/grabpl deploy-to-k8s',
```

Comment:
👍 . Yeah, it won't be needed anymore. Thought of modifying it instead of putting the new step directly in Drone. But the new step is a Drone plugin so it's easier to invoke it directly